src/inc/eventtracebase.h

```diff
@@ -216,11 +280,21 @@ extern "C" {
 #endif //!DONOT_DEFINE_ETW_CALLBACK && !DACCESS_COMPILE
 
 #include <clretwallmain.h>
+
+#elif defined(PLATFORM_UNIX)
```

Comment:
It is defined in the root cmakelist.txt at line 441:

add_definitions(-D__LINUX__=1)

This is being defined independent of PAL, so you have a way to distinguish Linux from other Unix OS.